pkg/sqs/testdata/sqs_source.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: numaflow.numaproj.io/v1alpha1
+kind: Pipeline
+metadata:
+  name: aws-sqs-source
+spec:
+  limits:
+    readBatchSize: 10 # for aws sqs the batch size should be between 1 -  10
+  vertices:
+    - name: in
+      scale:
+        min: 1
+      source:
+        udsource:
+          container:
+            image: "quay.io/numaio/numaflow-go/aws-sqs-source-go:latest"
+            env:
+              - name: QUEUE_NAME
+                value: "numaflow"
+              - name: REGION
+                value: "us-east-1"
+              - name: AWS_ACCESS_KEY
+                value: "AKIAS2LNCO4EY3JOZMTY"
+              - name: AWS_SECRET
+                value: "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"
```

Comment:
where do we get the access key and secret?

Reply:
the keys should be passed as environment variable ,anything else need to be done here ?